src/ontology/mondo-edit.obo

```diff
@@ -315079,7 +315454,8 @@ xref: Orphanet:1879 {source="MONDO:equivalentTo"}
 xref: UMLS:C2931505 {source="Orphanet:1879/e", source="MONDO:equivalentTo", source="Orphanet:1879"}
 xref: UMLS:C3149695 {source="MONDO:notFoundInDiseaseSubset", source="Orphanet:1879"}
 xref: UMLS:CN200621 {source="MONDO:equivalentTo"}
-is_a: MONDO:0008157 ! Buschke-Ollendorff syndrome
```

Comment:
@twhetzel yes. The class was unsatisfiable because it was both syndromic and isolated. This parent had no source, and based on the term definition and the review from Sylvia, I removed the "Buschke-Ollendorff syndrome" parent.